snmp/datadog_checks/snmp/data/default_profiles/aruba-wireless-controller.yaml

```diff
@@ -115,8 +115,8 @@ metrics:
           OID: 1.3.6.1.4.1.14823.2.2.1.1.3.3.1.12
         tag: ap_load_balancing
         mapping:
-          1: true
-          2: false
+          1: 'true'
+          2: 'false'
```

Comment:
new metric from in 7.48

Reply:
Fixed here https://github.com/DataDog/integrations-core/pull/15800/commits/d240f99058cd13c8581f0d4124506493ea22d354